packages/components/nodes/vectorstores/Chroma_Existing/Chroma_Existing.ts

```diff
@@ -69,7 +69,7 @@ class Chroma_Existing_VectorStores implements INode {
         const chromaURL = nodeData.inputs?.chromaURL as string
         const output = nodeData.outputs?.output as string
         const topK = nodeData.inputs?.topK as string
-        const k = topK ? parseInt(topK, 10) : 4
+        const k = topK ? parseFloat(topK) : 4
```

Comment:
these are `parseInt`. they are different from the `topK` used in models, although they have same name.

check with langchainjs code and API docs from models to see which one is decimal values and which one is integer

Reply:
oh nvm my bad, `parseFloat` also works